config/classifiers.yml

```diff
@@ -105,6 +112,7 @@ classifiers:
           stale_branches_count: 1
           life_period: 1
           last_changed: 1
+          questions_resolved_percent: 0.5
```

Comment:
Maturity is somewhat balanced grade of a project. That means:
 - it uses both metrics types (popularity and maintenance)
 - it use weights in more balanced way between overall activity and latest information.

So mature project should be popular and have enough maintenance quality.
Something like that.

As for future plans I want to add code and documentation quality metrics and all those metrics will contribute to final grade. 
Hope it'll clarify state a bit.